src/StatusQ/Popups/statusModal/StatusImageWithTitle.qml

```diff
@@ -64,12 +66,35 @@ Row {
                 width: statusImageWithTitle.image.width
                 height: statusImageWithTitle.image.height
                 color: statusImageWithTitle.image.isIdenticon ?
-                    Theme.palette.statusRoundedImage.backgroundColor :
-                    "transparent"
+                           Theme.palette.statusRoundedImage.backgroundColor :
+                           "transparent"
                 border.width: statusImageWithTitle.image.isIdenticon ? 1 : 0
                 border.color: Theme.palette.directColor7
                 showLoadingIndicator: true
             }
+
+            StatusRoundButton {
+                type: StatusRoundButton.Type.Secondary
+
+                anchors.right: parent.right
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: -3
+
+                width: 15
+                height: 15
+
+                border.color: Theme.palette.indirectColor1
+                border.width: 1
+
+                icon.name: "tiny/edit"
+                icon.width: 8
+                icon.height: 8
```

Comment:
Visually, this seems to be too tiny. It's possible that there's a padding white space in the icon source, so wie might need to bump up the width and height here a little.